.github/workflows/ci.yml

```diff
@@ -116,6 +116,20 @@ jobs:
         with:
           python-version: '3.8'
           architecture: 'x64'
+      - name: 'Cache Cargo home directory'
+        uses: actions/cache@v2
+        with:
+          path: |
+            tensorboard/data/server/target/
+            # https://doc.rust-lang.org/cargo/guide/cargo-home.html#caching-the-cargo-home-in-ci
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            # Needed for installing binaries with cache
```

Comment:
I don't quite understand this comment. Could you elaborate slightly?

Update: read the other version of this comment which mentions `cargo-raze` so I assume this has something to do with `raze` generating these artifacts?  Slightly more elaboration would still be useful IMO.

Reply:
Elaborated. This is needed only if you `cargo install` a binary, like
`cargo-raze` or `hyperfine` or something. Upstream documentation issue:
<https://github.com/rust-lang/cargo/issues/8841>

We don’t actually need it for this job. I kept it to reduce headache in
case we later update it to `cargo install`. Clarified in comment.